tests/datasource/data_connector/test_data_connector_util.py

```diff
@@ -15,6 +15,7 @@
     list_gcs_keys,
     map_batch_definition_to_data_reference_string_using_regex,
     map_data_reference_string_to_batch_definition_list_using_regex,
+    storage,
```

Comment:
Should we be importing the import from the source code or should we try to import GCS here?

Reply:
I purposely tried to import `storage` from the 3 different source files since those were the places where they were failing. The test skip reasons also reflect where storage was failed to be imported.